Source/DafnyCore/DooFile.cs

```diff
@@ -212,6 +221,30 @@ public class ManifestData {
     return false;
   }
 
+  // Checks that the library option ==> the local option.
```

Comment:
I think it's better to use `///` comments. IDE will then also move the comments if you move the definition.